src/bin/lib/subcommands.c/runprogram.h

```diff
@@ -188,17 +188,17 @@ execute_program(Program *prog)
 			/* fork succeeded, in child */
 
 			/*
-			 * We redirect /dev/null into stdin rather than closing stdin,
+			 * We redirect /dev/null into _stdin rather than closing stdin,
 			 * because apparently closing it may cause undefined behavior if
 			 * any read was to happen.
 			 */
-			int stdin = open(DEV_NULL, O_RDONLY);
+			int _stdin = open(DEV_NULL, O_RDONLY);
 
-			dup2(stdin, STDIN_FILENO);
+			dup2(_stdin, STDIN_FILENO);
```

Comment:
we don't use this naming schema. Can we use a standard naming like `stdIn` as we did om`struct Program`

Reply:
thanks, good point, fixed.